templates/matlab.yml

```
@@ -0,0 +1,44 @@
+# MATLAB
+# Run MATLAB and Simulink as part of your build pipeline.
+# This extension enables you to run MATLAB® scripts, functions, and statements as part of your build pipeline.
+# You also can run your MATLAB and Simulink® tests, generate artifacts such as JUnit test results and Cobertura code coverage reports, and publish your results to Azure Pipelines.
+# https://github.com/mathworks/matlab-azure-devops-extension/blob/master/overview.md
+
+trigger:
+- {{ branch }}
+
+pool:
+  {{ pool }}
+
+steps:
+  - task: InstallMATLAB@0
+    inputs:
+      release: latest # Replace the specific release name eg: R2021a, By default it picks the latest MATLAB version.
+
+# Uncomment this task to runs specific MATLAB command or script. eg : building your own mex files etc.
+#  - task: RunMATLABCommand@0
+#    inputs:
+#      command: myscript # Replace with appropriate MATLAB command or script file.
+
+
+# This task Runs all the MATLAB and Simulink tests and generate variety of test artifacts. Check the documentation for all supported task parameters.
+  - task: RunMATLABTests@0
+    inputs:
+      sourceFolder: mySrc;mySrc2 # Replace with your appropriate source folders names separated by ';' or ':'
+      testResultsPDF: test-results/results.pdf # Path to generate PDF test reports.
+      testResultsJUnit: test-results/results.xml # Path to generate Junit test report.
+      codeCoverageCobertura: code-coverage/coverage.xml # Path to generate Cobertura code coverage.
```

I think we should consider just showing `testResultsJUnit`. The PDF report feels redundant. The coverage report gets us into using `sourceFolder` which makes the simple template more complex than it probably needs to be.

---

Agreed. Currently, this is a copy of what we have in the README. There is room for simplification.

---

Whats the sugestion here , should we comment PDF report / code coerage & Source folder ?

---

My suggestion is just to get rid of them.
```suggestion
      testResultsJUnit: test-results/results.xml
```

---

Done.